src/core/public/rendering/app_containers.tsx

```diff
@@ -11,16 +11,31 @@ import { Observable } from 'rxjs';
 import useObservable from 'react-use/lib/useObservable';
 import classNames from 'classnames';
 
+export const APP_WRAPPER_CLASS = 'kbnAppWrapper';
+
 export const AppWrapper: React.FunctionComponent<{
   chromeVisible$: Observable<boolean>;
-}> = ({ chromeVisible$, children }) => {
+  classes$?: Observable<string[]>;
+}> = ({ chromeVisible$, classes$, children }) => {
   const visible = useObservable(chromeVisible$);
-  return <div className={classNames('app-wrapper', { 'hidden-chrome': !visible })}>{children}</div>;
+  const classes = useObservable(classes$, ['']);
```

Comment:
This line is throwing a TS error that I can't decipher because I'm not familiar with observables.

```
Argument of type 'Observable<string[]> | undefined' is not assignable to parameter of type 'Observable<string[]>'.
  Type 'undefined' is not assignable to type 'Observable<string[]>'.
```